src/lib/viewers/BaseViewer.js

```diff
@@ -560,12 +568,28 @@ class BaseViewer extends EventEmitter {
      * @return {void}
      */
     resize() {
+        this.isSmall = this.isSmallContainer();
+
+        if (this.isSmall) {
+            this.rootEl.classList.add(CLASS_BOX_PREVIEW_IS_SMALL);
+        } else {
+            this.rootEl.classList.remove(CLASS_BOX_PREVIEW_IS_SMALL);
+        }
+
         this.emit('resize', {
             width: document.documentElement.clientWidth,
             height: document.documentElement.clientHeight
         });
     }
 
+    /**
+     * Determines if the preview container is a small viewport
+     * @return {boolean} Whether the container is small
+     */
+    isSmallContainer() {
+        return this.options.container.clientWidth < BREAKPOINT_MEDIUM;
```

Comment:
Is `this.options.container` always defined and does it always include `clientWidth`?

Reply:
Yeah this appears to always be defined, if the user doesn't provide it, it defaults to `document.body`